src/services/indexer/dao/hooks/useDAO.ts

```diff
@@ -57,49 +57,32 @@ export const useDAO = (address: string) => {
           };
         }),
         type: dao.dao_type.name,
+        dao_type: dao.dao_type,
```

Comment:
There should be no need to save this parameter, given that DAOs are structured in a class hierarchy. Any specific change to either type should be encapsulated in each DAO subclass (Registry or Treasury)